### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ashish4321**
Added Explanation for Backslash at the end of string
Issue number #69

### CODE REVIEW

**Comment by ashish4321** [comment]
Pls check for issue number #69

**Comment by ashish4321** [comment]
Is it okay?

File: README.md [hunk 1]
```diff
@@ -662,7 +662,23 @@ SyntaxError: EOL while scanning string literal
   'wt\\"f'
   ```
 - What the interpreter actually does, though, is simply change the behavior of backslashes, so they pass themselves and the following character through. That's why backslashes don't work at the end of a raw string.
-
+- String quotes can be escaped with a backslash, but the backslash remains in the string; for example, r""" is a valid string literal consisting of two characters: a backslash and a double quote; r"" is not a valid string literal (even a raw string cannot end in an odd number of backslashes). Specifically, a raw string cannot end in a single backslash (since the backslash would escape the following quote character). Note also that a single backslash followed by a newline is interpreted as those two characters as part of the string, not as a line continuation.
+Few examples :
+
+>>> print(r"\"")
```
Comment (satwikkansal) [inline README.md]: hmm, the examples should be displayed like other examples in the repo. You can scroll through the readme to have an idea :)

File: README.md [hunk 2]
```diff
@@ -662,7 +662,23 @@ SyntaxError: EOL while scanning string literal
...
+\"
+>>> print(r"A\nB")
+A\nB
+>>> print(r"A\\nB")
```
Comment (satwikkansal) [inline README.md]: How about having an explanation after every triggering statement. Kind of like getting into more detailed explanation with each output. Hope you get it :)

File: README.md (hunk 1 above)
Comment (satwikkansal) [reply README.md]: huh, I didn't ask anyone anything to add..

**Pull request closed without merging**